sdk/typescript/src/module/introspector/typescript_module/typedef_utils.ts

```diff
@@ -3,17 +3,17 @@ import { IntrospectionError } from "../../../common/errors/index.js"
 import { TypeDef } from "../typedef.js"
 
 export function isTypeDefResolved(typeDef: TypeDef<TypeDefKind>): boolean {
-  if (typeDef.kind !== TypeDefKind.ListKind) {
+  if (typeDef.kind !== TypeDefKind.List) {
```

Comment:
Confused why this doesn't break :thinking: 

The `_KIND` values shouldn't be present before v0.18.11 - so how does this work for version compatibility on older versions? Why is this accessible?